scripts/zones/Selbina/IDs.lua

```diff
@@ -23,6 +23,8 @@ zones[xi.zone.SELBINA] =
         FISHING_MESSAGE_OFFSET        = 6565, -- You can't fish here.
         NOMAD_MOOGLE_DIALOG           = 6665, -- I'm a traveling moogle, kupo. I help adventurers in the Outlands access items they have stored in a Mog House elsewhere, kupo.
         SUBJOB_UNLOCKED               = 6868, -- You can now designate a support job.
+        FERRY_DEPARTING               = 7019, -- All aboard!
```

Comment:
Could you please put these additions in sequential order?

Reply:
done :)